DogServiceExample/src/main/java/com/common/rest/DogService.java

```diff
@@ -0,0 +1,110 @@
+package com.common.rest;
+
+import com.common.json.JsonHelper;
+import com.common.app.Dog;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.Response;
+import java.util.HashMap;
+import java.util.Map;
+
+
+@Path("/")
+public class DogService {
```

Comment:
This way you will not have to set status 200 manually, for example. And if you throw exceptions in Dog setters (if not passed validation) status will be set to 500 automatically.